pkg/urbit/vere/term.c

```diff
@@ -1213,19 +1213,19 @@ u3_term_io_hija(void)
     else {
       if ( c3n == u3_Host.ops_u.dem ) {
         if ( 0 != tcsetattr(1, TCSADRAIN, &uty_u->bak_u) ) {
-          perror("hija-tcsetattr-1");
+          u3l_log("term: hija tcsetattr-1: %s\r\n", strerror(errno));
```

Comment:
I suspect that the errors in the terminal itself need to be piped directly to stderr instead of trying to hijack the streams during an error.